README.md

```diff
@@ -7,8 +7,8 @@ This repository contains the list of communities and job portals you can join an
 ## Roadmaps:
 <p align="center"><img align="center" alt="Coding" src="https://media.giphy.com/media/xUySTOigOUHucl3rfW/giphy.gif"></p>
 <p  align="center"><a href="https://github.com/commclassroom/roadmaps/tree/main/Backend-Development#readme"><B>Backend-Development</B></a></p>
-<p  align="center"><a href=""><b>Blockchain</b></a></p>
-<p  align="center"><a href=""><b>DevOps</b></a></p>
+<!-- <p  align="center"><a href=""><b>Blockchain</b></a></p> -->
+<p  align="center"><a href="https://github.com/commclassroom/roadmaps/blob/main/DevOps/Readme.md"><b>DevOps</b></a></p>
```

Comment:
Please use relative linking and not the entire url. Please take a look at this [stack overflow website](https://stackoverflow.com/questions/7653483/github-relative-link-in-markdown-file)